rules.go

```diff
@@ -47,6 +48,9 @@ func init() {
 		if value == nil {
 			return err
 		}
+		if _, ok := value.(multipart.File); ok {
```

Comment:
@erickskrauch  What if the value is not file, a string or number? The required rules seems it will fail for other data validation.

Can you fix this and send PR against ***dev branch***

Reply:
I'm not sure I understood you correctly. [Below of this line](https://github.com/thedevsaddam/govalidator/blob/f3a2b50d2d05ad0ce4b1c4e395ba976db0c050b6/rules.go#L50-L148) you make validation of any other types by reflection. Can you provide me the case of the value, that will fail `required` validator?